gamedata/fbl/demon-generator.de.json

```diff
@@ -0,0 +1,286 @@
+{
+	"form": {
+		"roll": "D66",
+		"data": {
+			"11": {
+				"form": "Mensch",
+				"attributes": ["D6+3", 3, 3, 3],
+				"armor": "D6",
+				"symbol": "🙅‍"
+			},
+			"25": {
+				"form": "Minotaurus",
+				"attributes": ["D6+5", 3, 3, 3],
+				"armor": "D6",
+				"symbol": "🐮"
+			},
+			"31": {
+				"form": "Ziege",
+				"attributes": ["D6+3", 4, 3, 3],
+				"armor": "D6",
+				"symbol": "🐐"
+			},
+			"33": {
+				"form": "Bär",
+				"attributes": ["D6+6", 3, 3, 3],
+				"armor": "D6",
+				"symbol": "🐻"
+			},
+			"35": {
+				"form": "Schlange",
+				"attributes": ["D6+4", 4, 3, 3],
+				"armor": "D6",
+				"symbol": "🐍"
+			},
+			"41": {
+				"form": "Untoter",
+				"attributes": ["D6+3", 3, 3, 3],
+				"armor": "D6",
+				"symbol": "💀"
+			},
+			"43": {
+				"form": "Spinne",
+				"attributes": ["D6+5", 4, 3, 3],
+				"armor": "D6",
+				"symbol": "🕷"
+			},
+			"45": {
+				"form": "Giftschlange",
+				"attributes": ["D6+3", 4, 3, 3],
+				"armor": "D6",
+				"symbol": "🐍"
+			},
+			"51": {
+				"form": "Wolf",
+				"attributes": ["D6+4", 4, 3, 3],
+				"armor": "D6",
+				"symbol": "🐺"
+			},
+			"53": {
+				"form": "Insekt",
+				"attributes": ["D6+2", 3, 3, 3],
+				"armor": "D6",
+				"symbol": "🐞"
+			},
+			"55": {
+				"form": "Wolfsmensch",
+				"attributes": ["D6+4", 4, 3, 3],
+				"armor": "D6",
+				"symbol": "🐺"
+			},
+			"61": {
+				"form": "Riesenkrabbe",
+				"attributes": ["D6+5", 2, 3, 3],
+				"armor": "D6+3",
+				"symbol": "🦀"
+			},
+			"63": {
+				"form": "Riesenkröte",
+				"attributes": ["D6+5", 2, 3, 3],
+				"armor": "D6",
+				"symbol": "🐸"
+			},
+			"64": {
+				"form": "Krake",
+				"attributes": ["D6+4", 2, 3, 3],
+				"armor": "D6",
+				"effect": "Rennen ist an Land eine langsame Aktion.",
+				"symbol": "🦑"
+			},
+			"65": {
+				"form": "Nebel",
+				"attributes": ["D6+3", 3, 3, 3],
+				"armor": 0,
+				"effect": "Kann von physischen Waffen nicht verwundet werden.",
+				"symbol": "👻"
+			},
+			"66": {
+				"form": "Schatten",
+				"attributes": ["D6+3", 3, 3, 3],
+				"armor": 0,
+				"effect": "Kann von physischen Waffen nicht verwundet werden.",
+				"symbol": "👻"
+			}
+		}
+	},
+	"abilities": {
+		"roll": "D66",
+		"array": true,
+		"data": {
+			"11": ["Steinhaut", "Rüstungsschutz +6"],
+			"13": ["Riesig", "Stärke +2W6"],
```

Comment:
```suggestion
			"13": ["Riesig", "Stärke +2D6"],
```

Reply:
I changed all the dice references to support the english "D"-format. I think you're right and this is best to have one abbreviation for all.